commit 3fd8f90f4c9b9825d832e00e57e1b5fefce1e058
Author: LeeDrOiD <leedroid.lb@gmail.com>
Date:   Sun May 15 18:00:40 2011 +0100

    arch/arm/mm/alignment: fix warning in alignment fixup [Jamie Iles]

diff --git a/arch/arm/mm/alignment.c b/arch/arm/mm/alignment.c
index 6f98c35..28f100e 100644
--- a/arch/arm/mm/alignment.c
+++ b/arch/arm/mm/alignment.c
@@ -826,6 +826,7 @@ do_alignment(unsigned long addr, unsigned int fsr, struct pt_regs *regs)
 			handler = do_alignment_t32_to_handler(&instr, regs, &offset);
 		else
 			handler = do_alignment_ldmstm;
+			offset.un = 0;
 		break;
 
 	default:
